### PR TITLE
Add `Debug Media`'s playlists to malayalam Git, HTML, CSS, React sections

### PR DESCRIPTION
**Pull request by GH05T-HUNTER5**
New Update

Added React , HTML and CSS , Git

### CODE REVIEW

**Review by davorpa — CHANGES_REQUESTED** [review]
It's time to make some minor changes to follow the formatting of listing resources.

Take your time to apply the following suggestions over the same branch where this PR was opened: `patch-11`

Any other intent of submitting incremental PRs will be considered as spam.

**Comment by davorpa** [comment]
https://github.com/EbookFoundation/free-programming-books/pull/7004#discussion_r945411837 suggestion is not applied yet. 

Why did you resolve the conversation? I'd unhided it again

**Comment by davorpa** [comment]
https://github.com/EbookFoundation/free-programming-books/pull/7004#discussion_r945411837 suggestion is not applied yet. 

Why did you resolve the conversation? I'd unhided it again

**Comment by davorpa** [comment]
https://github.com/EbookFoundation/free-programming-books/pull/7004#discussion_r945411837 suggestion is not applied yet. 

Why did you resolve the conversation? I'd unhided it again

**Comment by GH05T-HUNTER5** [comment]
> 

Sorry

**Comment by GH05T-HUNTER5** [comment]
> [#7004 (comment)](https://github.com/EbookFoundation/free-programming-books/pull/7004#discussion_r945411837) suggestion is not applied yet.
> 
> Why did you resolve the conversation? I'd unhided it again

That's my friend (sry)

**Comment by GH05T-HUNTER5** [comment]
> [#7004 (comment)](https://github.com/EbookFoundation/free-programming-books/pull/7004#discussion_r945411837) suggestion is not applied yet.
> 
> Why did you resolve the conversation? I'd unhided it again

- added

File: courses/free-courses-ml.md [hunk 1]
```diff
@@ -88,6 +91,7 @@
 * [React js malayalam tutorial](https://youtube.com/playlist?list=PLBKJGko2JEdG8FUKu2hUF6K7irvbT8hIM) - Tintu Vlogger
 * [React Challenge](https://youtube.com/playlist?list=PLY-ecO2csVHfgVM9sChmUirqK7BXUBX9P) - Brototype Malayalam
 * [React JS - Malayalam Tutorial - by Yes Tech Media](https://youtube.com/playlist?list=PL1JrLEBAapUX44d5zzl0hf7vX7caSCidT) - Yes Tech Media
+* [React js malayalam tutorial](https://youtube.com/playlist?list=PL4KwFGqvN4nuKJ3bJSP-LcchabWJtPtE8) Debug Media
```
[the remark this answers is not in the document]
Comment (davorpa) [reply courses/free-courses-ml.md]: Fixed by commit 9cef8c2793e9cc2c97d9ab33732ca3390e2a4edd

**Pull request merged**